src/plugins/recordTypes/nagprainventory/fields.js

```diff
@@ -0,0 +1,775 @@
+              },
+            },
+            involvedRole: {
+              [config]: {
+                messages: defineMessages({
+                  fullName: {
+                    id: 'field.nagprainventories_common.involvedRole.fullName',
+                    defaultMessage: 'Parties involved role',
+                  },
+                  name: {
+                    id: 'field.nagprainventories_common.involvedRole.name',
+                    defaultMessage: 'Role',
+                  },
+                }),
+                view: {
+                  type: TermPickerInput,
+                  props: {
+                    source: 'nagprainvolvedrole',
+                  },
+                },
+              },
+            },
+          },
+        },
+        culturalAffiliationGroupList: {
+          [config]: {
+            view: {
+              type: CompoundInput,
+            },
+          },
+          culturalAffiliationGroup: {
+            [config]: {
+              messages: defineMessages({
+                name: {
+                  id: 'field.nagprainventories_common.culturalAffiliationGroup.name',
+                  defaultMessage: 'Inventory cultural affiliation',
+                },
+              }),
+              repeating: true,
+              view: {
+                type: CompoundInput,
+                props: {
+                  tabular: false,
+                },
+              },
+            },
+            tribeOrNation: {
+              [config]: {
+                messages: defineMessages({
+                  fullName: {
+                    id: 'field.nagprainventories_common.tribeOrNation.fullName',
+                    defaultMessage: 'Inventory cultural affiliation tribe/nation',
+                  },
+                  name: {
+                    id: 'field.nagprainventories_common.tribeOrNation.name',
+                    defaultMessage: 'Tribe/nation',
+                  },
+                }),
+                view: {
+                  type: AutocompleteInput,
+                  props: {
+                    source: 'organization/local,organization/ulan',
+                  },
+                },
+              },
+            },
+            includeInNotice: {
+              [config]: {
+                messages: defineMessages({
+                  fullName: {
+                    id: 'field.nagprainventories_common.includeInNotice.fullName',
+                    defaultMessage: 'Inventory cultural affiliation include in notice',
+                  },
+                  name: {
+                    id: 'field.nagprainventories_common.includeInNotice.name',
+                    defaultMessage: 'Include in notice',
+                  },
+                }),
+                view: {
+                  type: TermPickerInput,
+                  props: {
+                    source: 'nagpranotice',
+                  },
+                },
+              },
+            },
+            determinedBy: {
```

Comment:
This needs to be updated to be repeatable to be in line with the spec. Likely also needs to be fixed in duty of care.